pkg/monitor/cluster/clustersync.go

```diff
@@ -78,3 +76,8 @@ func timeToString(t *metav1.Time) string {
 	}
 	return t.String()
 }
+
+func cleanString(s string) string {
+	// remove special character and new line except _ and -
+	return regexp.MustCompile(`[^\w\-\n]`).ReplaceAllString(s, "")
```

Comment:
MustCompile is better used in global vars because it panics when a regex is invalid, when a panic happens it will be detected right away during testing/development. Although I can see that the regex here is not invalid so there is low risk. I think a better use here is Compile where it returns an error and that error must be checked.